modules/local/eukulele/main.nf

```diff
@@ -31,7 +31,7 @@ process EUKULELE {
 
     rc=0
     mkdir contigs
-    mv $input ./contigs/
+    cp $input ./contigs/
```

Comment:
Let's remember to check this -- can it take a lot of space?

Reply:
yes. I changed it before from ´cp´to ´mv´ but then I decided to restore the previous code because I was worried that with ´mv´ the module is not able to catch a second time the ´$input´ ( I experienced this error but maybe there was something else).